# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/SharedSurfaceGL.cpp
# Commit: 7e2a2b1b416f
# Full Hash: 7e2a2b1b416f1a64c6548aab29e92acc9554ae7d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix all webgl regression tests according to CI. r=handyman
#   
#   (This is a combination of 31 commits)
#   
#   * Fix Linux compilation.
# ==============================================================================

diff -r ccfa767dba64 -r 7e2a2b1b416f gfx/gl/SharedSurfaceGL.cpp
--- a/gfx/gl/SharedSurfaceGL.cpp	Wed Jan 08 22:19:16 2020 +0000
+++ b/gfx/gl/SharedSurfaceGL.cpp	Wed Jan 08 22:19:23 2020 +0000
@@ -9,6 +9,7 @@
 #include "GLContext.h"
 #include "GLReadTexImageHelper.h"
 #include "mozilla/gfx/2D.h"
+#include "mozilla/layers/TextureForwarder.h"
 #include "ScopedGLHelpers.h"
 
 namespace mozilla {